pkg/base1/test-metrics.js

```diff
@@ -104,6 +104,16 @@ function MockSink(expected, callback) {
     return self;
 }
 
+let old_channel;
+QUnit.module("Mock tests", {
+    beforeEach: () => {
+        old_channel = cockpit.channel;
+    },
+    afterEach: () => {
+        cockpit.channel = old_channel;
```

Comment:
Why is that necessary? AFAIK it does not get diverted anywhere?

Reply:
It does in mockChannel.

Reply:
Oh, you mean an existing test damages this, and for the new ones we want to use the real cockpit.channel.